games/views.py

```diff
@@ -263,11 +263,19 @@ def add_player(request):
         'role_choices': Player.ROLE_CHOICES,
     }
     return render(request, 'games/add_player.html', context)
+
 @login_required
 def check_username(request):
```

Comment:
the function name does not reflect what it does (after the adding the email check)